flags/flags.go

```diff
@@ -195,6 +195,21 @@ $STEPPATH/config/ca.json`,
 be stored in the 'x5c' header.`,
 	}
 
+	// X5tCert is a cli.Flag used to pass the x5t header certificate thumbprint
+	// for a JWS or JWT.
+	X5tCert = cli.StringFlag{
+		Name:  "x5t-cert",
+		Usage: "Certificate in PEM format to use for the 'x5t' header of a JWS or JWT",
+	}
```

Comment:
We should have a `<path>` or something tagged with `<>` so the help displays `x5t-cert=path` instead of `x5t-cert=value`